src/routes.ts

```diff
@@ -25,13 +27,15 @@ export async function routes(fastify: FastifyInstance): Promise<void> {
   fastify.route({
     method: "GET",
     url: "/transfers/:id",
+    schema: transferByIdSchema,
     handler: TransfersController.transferById,
   })
 
   fastify.route({
     method: "GET",
-    url: "/transfers/txHash/:txHash",
-    handler: TransfersController.transferByTxHash,
+    url: "/transfers/txHash/:txHash/domains/:domainID",
```

Comment:
Considering Matija consern, let's go with `/transfers/txHash/:txHash?domainID=${domainID}` In that way we won't break the current implementation of explorer-ui, and we give possibility to filter it also by domainID @mj52951 @wainola

Reply:
Agree, will update the SDK PR that is going to use this route